build.gradle.kts

```diff
@@ -135,10 +140,25 @@ allprojects {
 
         input = files("$rootDir/buildSrc", "build.gradle.kts", "src/main/kotlin", "src/test/kotlin",
             "src/funTest/kotlin")
+
+        basePath = rootProject.projectDir.path
+
+        reports {
+            html.enabled = false
+            sarif.enabled = true
```

Comment:
I guess generating the report doesn't take much time, but what about setting this to false by default and enabling it with a property, just to make the task on the command line as fast as possible?

Reply:
This would make the task logic more complex in order to conditionally *not* look for SARIF files in the merge tasks which finalize the detekt task now, so I'd rather not do that micro optimization.